fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/FluidView.java

```diff
@@ -0,0 +1,121 @@
+package net.fabricmc.fabric.api.fluids.v1;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Iterators;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.Entity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Box;
+import net.minecraft.util.math.Direction;
+import net.minecraft.world.World;
+
+import net.fabricmc.fabric.api.fluids.v1.container.FluidContainer;
+import net.fabricmc.fabric.api.fluids.v1.container.SimpleFluidContainer;
+import net.fabricmc.fabric.api.fluids.v1.container.volume.ImmutableFluidVolume;
+import net.fabricmc.fabric.api.fluids.v1.item.ItemFluidContainer;
+import net.fabricmc.fabric.api.fluids.v1.item.ItemSink;
+import net.fabricmc.fabric.api.fluids.v1.math.Drops;
+import net.fabricmc.fabric.api.fluids.v1.world.SidedFluidContainer;
+
+/**
+ * A util class for accessing fluid containers.
+ *
+ * @see Drops
+ */
+public class FluidView {
```

Comment:
could possibly merge these into one helper class or name this class to something like FluidViewHelper